README.md

```diff
@@ -146,10 +324,11 @@ The URLs to download the data are constructed according to the following pattern
 
 - {datasetname} : In the URLs the dataset name must follow the ML naming (`training-input` or `training-output-reference` or `training-output-benchmark`).
 - {format} is `netcdf`. Training output is also available as GRIB file,  using `format='grib'` and replacing `".nc"` by `".grib"`
+- {fctype}: `hindcast` for training or `forecast` for test
 - {parameter} is `t2m` for [surface temperature at 2m](https://confluence.ecmwf.int/display/S2S/S2S+Surface+Air+Temperature), `tp` for [total precipitation](https://confluence.ecmwf.int/display/S2S/S2S+Total+Precipitation)
-- {origin} : `ecmwf` or `eccc` or `ncep` <!-- we should have a clean table for this once -->
+- {origin} : `ecmwf` or `eccc` or `ncep`
 - {weeks} from [`"34"`, `"56"`, `["34", "56"]`] only for `benchmark`
-- `YYYYMMDD` is the date of main forecast time in the file.
+- `YYYYMMDD` is the date of the 2020 forecast for `test-input`/`forecast-input`. The same dates are required for the on-the-fly `training-input`/`hindcast-input` but return the multi-year hindcast for that `MMDD`.
```

Comment:
Btw, we could add the possibility to do this in the plugin code:
cml.load_dataset("training-input", date="0201")
cml.load_dataset("training-input", date=slice("0101","0201" )
cml.load_dataset("training-input", date="01*")

Reply:
Agree. Probably easier to understand.

Reply:
moved to issue. very low priority